app/developers/deliver_routes.py

```diff
@@ -952,11 +951,38 @@ def list_submissions_for_collection(collection_id: UUID, submission_mode: Submis
         ),
         grant=collection.grant,
         collection=collection,
-        submissions=submissions,
+        submissions=[submission for _, submission in helper.submission_helpers.items()],
+        submission_mode=submission_mode,
         is_test_mode=submission_mode == SubmissionModeEnum.TEST,
     )
 
 
+@developers_deliver_blueprint.route(
+    "/collections/<uuid:collection_id>/submissions/<submission_mode:submission_mode>/export",
+    methods=["GET"],
+)
+@is_platform_admin
+def export_submissions_for_collection(collection_id: UUID, submission_mode: SubmissionModeEnum) -> ResponseReturnValue:
+    export_format = request.args.get("format", "")
```

Comment:
```suggestion
@developers_deliver_blueprint.route(
    "/collections/<uuid:collection_id>/submissions/<submission_mode:submission_mode>/export/<export_format>",
    methods=["GET"],
)
@is_platform_admin
def export_submissions_for_collection(collection_id: UUID, submission_mode: SubmissionModeEnum, export_format: str) -> ResponseReturnValue:
```

optional nit: could embed the export type in the URL so it's a bit more discoverable than a random query arg?